src/python/WMComponent/AnalyticsDataCollector/AnalyticsPoller.py

```diff
@@ -77,6 +78,12 @@ def setup(self, parameters):
             pluginFactory = WMFactory("plugins", "WMComponent.AnalyticsDataCollector.Plugins")
             self.plugin = pluginFactory.loadObject(classname=self.pluginName)
 
+        # delete all agent-request type doc.
+        self.localSummaryCouchDB.deleteAllAgentRequestDocument()
```

Comment:
Seangchan, what happens to the job-related documents? Can they be recreated and wmstats repopulated with them?